core/src/main/java/com/klaytn/caver/transaction/type/EthereumAccessList.java

```diff
@@ -0,0 +1,541 @@
+        }
+
+        this.fillTransaction();
+
+        String hash = signer.apply(this);
+        SignatureData signatureData = keyring.ecsign(hash, AccountKeyRoleBased.RoleGroup.TRANSACTION.getIndex(), index);
+
+        this.appendSignatures(signatureData);
+
+        return this;
+    }
+
+    /**
+     * Getter function for to
+     *
+     * @return String
+     */
+    public String getTo() {
+        return to;
+    }
+
+    /**
+     * Setter function for to
+     *
+     * @param to The account address that will receive the transferred value.
+     */
+    public void setTo(String to) {
+        // "to" field in EthereumAccessList allows null
+        if (to == null || to.isEmpty()) {
+            to = "0x";
+        }
+
+        if (!to.equals("0x") && !Utils.isAddress(to)) {
+            throw new IllegalArgumentException("Invalid address. : " + to);
+        }
+        this.to = to;
+    }
+
+    /**
+     * Getter function for input
+     *
+     * @return String
+     */
+    public String getInput() {
+        return input;
+    }
+
+    /**
+     * Setter function for input
+     *
+     * @param input Data attached to the transaction.
+     */
+    public void setInput(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException("input is missing");
+        }
+
+        if (!Utils.isHex(input)) {
+            throw new IllegalArgumentException("Invalid input : " + input);
+        }
+        this.input = Numeric.prependHexPrefix(input);
+    }
+
+
+    /**
+     * Getter function for value
+     *
+     * @return String
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Setter function for value
+     *
+     * @param value The amount of KLAY in peb to be transferred.
+     */
+    public void setValue(String value) {
+        if (value == null) {
+            throw new IllegalArgumentException("value is missing");
+        }
+
+        if (!Utils.isNumber(value)) {
+            throw new IllegalArgumentException("Invalid value : " + value);
+        }
+        this.value = value;
+    }
+
+    /**
+     * Setter function for value
+     *
+     * @param value The amount of KLAY in peb to be transferred.
+     */
+    public void setValue(BigInteger value) {
+        setValue(Numeric.toHexStringWithPrefix(value));
+    }
+
+    /**
+     * Getter function for accessList
+     *
+     * @return accessList Access list is an EIP-2930 access list.
+     */
+    public AccessList getAccessList() {
+        return accessList;
+    }
+
+    /**
+     * Setter function for accessList
+     *
+     * @param accessList Access list is an EIP-2930 access list.
+     */
+    public void setAccessList(AccessList accessList) {
+        if (accessList == null) {
+            accessList = new AccessList();
+        }
+        this.accessList = accessList;
+    }
+}
```

Comment:
Don't we need to overloaded getTransactionHash() and getSenderTxHash()?

Reply:
[getTransactionHash](https://github.com/klaytn/caver-java/pull/291/files/dcc64baba4ab49c57000d729b3e4ea600c67dc3d#diff-7d2e50f6f873b201dbc7a152e444553518665448486ce1b968a245aac93d4872R356-R363) is already overrided.

We don't need to override getSenderTxHash because Abstract.getSenderTxHash call instance.getTransactionHash internally.